packages/expo-local-authentication/src/LocalAuthentication.ts

```diff
@@ -30,7 +30,8 @@ export async function isEnrolledAsync(): Promise<boolean> {
 }
 
 export async function authenticateAsync(
-  promptMessageIOS: string = 'Authenticate'
+  promptMessageIOS: string = 'Authenticate',
```

Comment:
What would you say if we move this `promptMessageIOS` into options object? This will be a breaking change, but that would be much better. Let's call it just `promptMessage` then.

Reply:
Yes, I was thinking of it as well.
I need this change to be available in Expo 33 and I am not sure it is doable if we introduce breaking change

Reply:
Maybe we can merge this first for Exo 33 and then introduce breaking change for Expo 34

Reply:
@tsapeta DO NOT merge this PR. I think I find a bug in the code